apps/condo/domains/organization/schema/Organization.test.js

```diff
@@ -1,92 +1,142 @@
+    // `registerNewOrganization` GraphQL mutation, that creates all the stuff without
+    // access check, using `execGqlWithoutAccess` under the hood.
+    test('user: create Organization', async () => {
+        const userClient = await makeClientWithNewRegisteredAndLoggedInUser()
+        try {
+            await createTestOrganization(userClient)
+        } catch (e) {
+            expect(e.errors[0]).toMatchObject({
+                'message': 'You do not have access to this resource',
+                'name': 'AccessDeniedError',
+                'path': ['obj'],
+            })
+            expect(e.data).toEqual({ 'obj': null })
+        }
     })
 
-    test('anonymous: no access to count', async () => {
+    test('anonymous: create Organization', async () => {
         const client = await makeClient()
-        const { data, errors } = await Organization.count(client, {}, { raw: true })
-        expect(errors[0]).toMatchObject({
-            'data': { 'target': '_allOrganizationsMeta', 'type': 'query' },
-            'message': 'You do not have access to this resource',
-            'name': 'AccessDeniedError',
-            'path': ['meta', 'count'],
-        })
-        expect(data).toEqual({ meta: { count: null } })
+        try {
+            await createTestOrganization(client)
+        } catch (e) {
+            expect(e.errors[0]).toMatchObject({
+                'message': 'You do not have access to this resource',
+                'name': 'AccessDeniedError',
+                'path': ['obj'],
+            })
+            expect(e.data).toEqual({ 'obj': null })
+        }
     })
 
-    test('user: allow to getAll', async () => {
+    test('user: read Organization — only those, it employed in', async () => {
         const admin = await makeLoggedInAdminClient()
-        await createOrganization(admin)
-        const [, userAttrs] = await createTestUser(admin)
-        const client = await makeLoggedInClient(userAttrs)
+        await createTestOrganization(admin)
+        const client = await makeClientWithRegisteredOrganization()
+
         const objs = await Organization.getAll(client, {})
-        expect(objs.length).toBeGreaterThan(0)
+        expect(objs.length).toBe(1)
+        expect(objs[0].id).toEqual(client.organization.id)
+    })
+
+    test('anonymous: read Organization', async () => {
+        const client = await makeClient()
+
+        try {
+            await Organization.getAll(client)
+        } catch (e) {
+            expect(e.errors[0]).toMatchObject({
+                'message': 'You do not have access to this resource',
+                'name': 'AccessDeniedError',
+                'path': ['objs'],
+            })
+            expect(e.data).toEqual({ 'objs': null })
+        }
+    })
+
+    // Test case, where user can update organization are in OrganizationEmployeeRole.test.js
+    test('user: update Organization — cannot', async () => {
```

Comment:
two cases! 1) update created organization
2) update another user organization

Reply:
Fixed in `bd7a6407`